packages/headless/src/controllers/facets/facet-search/facet-search.test.ts

```diff
@@ -107,6 +120,24 @@ describe('FacetSearch', () => {
     });
   });
 
+  describe('#clear', () => {
+    beforeEach(() => {
+      const options = buildMockFacetSearchRequestOptions();
+      engine.state.facetSearchSet[facetId] = buildMockFacetSearch({
+        options,
+      });
+      controller.clear();
+    });
+
+    it('#clear dispatches #clearFacetSearch', () => {
+      const clearFacetSearchAction = clearFacetSearch({
+        facetId,
+      });
+
+      expect(engine.actions).toContainEqual(clearFacetSearchAction);
+    });
```

Comment:
Since there is only one `it` block, it would be possible to combine the setup and remove the `describe` and `beforeEach` blocks.